examples/misc_controls_orbit.html

```diff
@@ -72,9 +73,10 @@
 				camera.position.z = 500;
 
 				controls = new THREE.OrbitControls( camera, renderer.domElement );
-				//controls.addEventListener( 'change', render ); // add this only if there is no animation loop (requestAnimationFrame)
-				controls.enableDamping = true;
-				controls.dampingFactor = 0.25;
+				controls.addEventListener( 'change', render ); // remove when using animation loop
+				// enable animation loop when using damping or autorotation
```

Comment:
> You're not going to automatically start an animation loop whenever the OrbitControls are used, are you?

Yes I am. But it will dispatch `change` only when needed (by comparing previous frame matrix vs current matrix).